_config.yml

```diff
@@ -102,7 +102,7 @@ ignore:
 # Extensions
 ## Plugins: https://hexo.io/plugins/
 ## Themes: https://hexo.io/themes/
-theme: geek
+theme: Nlvi
```

Comment:
Is this part of the PR?

Reply:
I think that's a mistake I should fix

Reply:
BTW I'd be happy to try another theme, so if you want to move it to another PR I'd be good with that.

Reply:
Yes, the responsiveness of the geek theme is quite terrible.

I'll create an issue